Week-5/Assignment-5.1/README.md

```diff
@@ -0,0 +1,82 @@
+# Assignment-5.1
+
+Using Async/Await and Generators, create separate functions and achieve the same functionality. Execute 3 callback functions asynchronously, for example doTask1(), doTask2() and doTask3().
+
+## Solution
+
+```sh
+// 3 asynchronous callback functions
+
+function doTask1(val) {
```

Comment:
is this solution or something else?

Reply:
Actually we had to create 3 tasks that would run using using async/await and generators